frappe/desk/query_report.py

```diff
@@ -530,7 +530,7 @@ def get_data_for_custom_report(columns):
 
 
 @frappe.whitelist()
-def save_report(reference_report, report_name, columns):
+def save_report(reference_report, report_name, json):
```

Comment:
json is a bad choice for fieldname, it is a standard python module

Reply:
#15708